.github/workflows/promote-image-to-env.yml

```diff
@@ -0,0 +1,93 @@
+#This workflow requires an input and based on that input pulls the appropriate image and tags it with the next environments name
+#This workflow will also likely change as we get more of our environment set up
+
+name: Promote an image to test or prod
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Select the environment
+        required: true
+        options:
+        - test
+        - prod
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+    environment: 
+      name: Testing
+    if: ${{ github.event.inputs.environment == 'test' }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cluster Login
+        uses: redhat-developer/openshift-actions@v1.1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT4_SERVER_URL }}
+          parameters: '{"apitoken": "${{ secrets.OPENSHIFT4_SA_PASSWORD }}"}'
+          cmd: |
+            'version'
+
+      - name: Login to Openshift Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.OPENSHIFT4_REGISTRY }}
+          username: ${{ secrets.OPENSHIFT4_SA_USERNAME }}
+          password: ${{ secrets.OPENSHIFT4_SA_PASSWORD }}
+
+      - name: Pull Dev Image 
+        run: |  
+          docker pull ${{ secrets.OPENSHIFT4_REGISTRY }}/${{ secrets.LICENSE_PLATE }}-tools/cccm-api:dev    
```

Comment:
At some point we may want to separate our promotions by service, but today is not that day :D Promoting everything at once is totally fine for now :)

Reply:
yeah I have been thinking about that too! Kind of not necessary to pull and push each one every time. Ill make a ticket for it :)